### DISCUSSION TITLE
🚀 Road to v1

### DISCUSSION THREAD

**Discussion by jycouet** [jycouet/kitql#20] [General]
## 🚀 Road to v1

Here it's what I have in mind for now:

- [x] https://github.com/jycouet/kitql/issues/28
- [x] https://github.com/jycouet/kitql/issues/9
- [x] https://github.com/jycouet/kitql/issues/39
- [ ] https://github.com/jycouet/kitql/issues/7
- [ ] https://github.com/jycouet/kitql/issues/8
- [ ] https://github.com/jycouet/kitql/issues/37
- [ ] https://github.com/jycouet/kitql/issues/38

If you have any suggestions, I would be happy to add them now or for later 💯

**Comment by draylegend** [comment]
Subscriptions, live queries or http2 stream something from this would be awesome!

Maybe later it's worth it to add defer.

Thank you for your effort and time 🙏🏻

**Reply by jycouet** [reply]
I'll have a look at all this ;)

**Reply by frederikhors** [reply]
I think this is not the top priority. We should first make basic things work WELL.

**Reply by joshua1** [reply]
Subscription is a priority, if not the others. Is a key graphql component in use a lot and a lib like this will Help a lot of subscription is first class, updating subscription stores

**Reply by jycouet** [reply]
Of course, I plan to work on this. It's a good topic that needs to be addressed.
I don't have the time ATM but will look into it one day 

Star / Follow / PR, everything is welcome as well 👍

**Reply by gevera** [reply]
> Subscriptions, live queries or http2 stream something from this would be awesome!
> 
> Maybe later it's worth it to add defer.
> 
> Thank you for your effort and time 🙏🏻

This is one for me is of top prority. Svelte stores already have subsctiptions, and I've used ruql before with websocket transport to have live updates on graphql subscriptions. It would definetely make KitQL even more awesome with this feature! 100%
